### PR TITLE
Loop through encoder buffer sizes to reach certain failure cases

### PR DESCRIPTION
**Pull request by sdpetrides**
By slowing incrementing the buffer sizes for each QIF header, the encoder reaches many more of failure cases.

### CODE REVIEW

File: bin/interop-encode.c [hunk 1]
```diff
@@ -319,7 +319,7 @@ main (int argc, char **argv)
         }
     }
 
-    if (0 != lsqpack_enc_init(&encoder, stderr,  dyn_table_size,
+    if (0 != lsqpack_enc_init(&encoder, NULL,  dyn_table_size,
```
Comment (dtikhonov) [inline bin/interop-encode.c]: Is this change related?

Comment (sdpetrides) [reply bin/interop-encode.c]: It is not related but running the tests takes ~5 minutes with logging enabled.

Comment (dtikhonov) [reply bin/interop-encode.c]: Maybe add a flag to run in "coverage" mode and have it off by default?  Because as a standalone tool, it's mostly useful for debugging and that's where logs are useful.

Comment (sdpetrides) [reply bin/interop-encode.c]: I see. Can I match the pattern for _interop-decode.c_ where it can be specified by command line and is off for testing?

Comment (dtikhonov) [reply bin/interop-encode.c]: Sure.
